guides/common/modules/proc_configuring-for-uefi-http-boot-ipv6-provisioning.adoc

```diff
@@ -25,5 +25,5 @@ ifndef::foreman-deb[]
 ----
 endif::[]
 ifdef::katello,orcharhino,satellite[]
-. Synchronize the {EL} 8 kickstart repository.
+. Synchronize the {EL} 9 or {EL} 8 kickstart repository.
```

Comment:
I think this one is unrelated to the server running on EL9.

Reply:
Addressed :+1: